Automattic/wp-calypso
# Rename concierge to quick start

#### Changes proposed in this Pull Request

* Rename all occurrences of `concierge` to `Quick Start`, per p9jf6J-1Zs-p2.
* Add refund policy for no-shows in the Quick Start session landing page and Support Session landing page.

#### Testing instructions

Verify that the text is "Quick Start" in the following pages:

1. In concierge booking page `/me/concierge`

<img width="769" alt="Screenshot 2019-09-05 at 6 45 05 PM" src="https://user-images.githubusercontent.com/1269602/64345096-50c8cd00-d00d-11e9-8b78-89cf129f3e18.png">

2. If you have multiple sites in your account, then `/me/concierge` site picker should say Quick Start:

<img width="478" alt="Screenshot 2019-09-05 at 6 32 13 PM" src="https://user-images.githubusercontent.com/1269602/64344272-c5027100-d00b-11e9-91b9-8a2b84c6904c.png">

3. In `/plans/my-plan/SITE_SLUG` in a business plan

<img width="1315" alt="Screenshot 2019-09-05 at 6 27 59 PM" src="https://user-images.githubusercontent.com/1269602/64344554-4eb23e80-d00c-11e9-9354-696d09c4ee46.png">

4. In the NPS Survey(you need to force the nps survey in your sandbox)

<img width="1127" alt="Screenshot 2019-09-05 at 6 43 13 PM" src="https://user-images.githubusercontent.com/1269602/64345021-2ecf4a80-d00d-11e9-946e-35f73e7efbee.png">

5. Thank You page for Business Plan purchase

<img width="690" alt="Screenshot 2019-09-05 at 8 44 48 PM" src="https://user-images.githubusercontent.com/1269602/64355667-3c8dcb80-d01f-11e9-8032-6ac8047b3276.png">

6. The banner displayed for Business plan/eCommerce plan sites on /me/purchases:

<img width="1483" alt="Screenshot 2019-10-08 at 1 08 09 PM" src="https://user-images.githubusercontent.com/1269602/66376257-d6ed8000-e9cc-11e9-863b-1e40b513d07a.png">
